src/xref_runner.erl

```diff
@@ -85,7 +100,9 @@ check(Check, Config) ->
 
     FilteredResults = filter_xref_results(Check, Results),
 
-    [result_to_warning(Check, Result) || Result <- FilteredResults]
+    ResultToPrint = [result_to_warning(Check, Result) || Result <- FilteredResults],
```

Comment:
Line 103 is too long:     ResultToPrint = [result_to_warning(Check, Result) || Result <- FilteredResults],.